src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block{
+	private String value;
+	
+	public Block(String value){
+		this.value = value;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.